packages/extension-chakra-storefront/config/default.json

```diff
@@ -1,50 +1,143 @@
 {
     "enabled": true,
     "activeDataEnabled": false,
+    "maxCacheAge": 900,
+    "staleWhileRevalidate": 900,
     "commerceAPI": {
         "proxyPath": "/mobify/proxy/api",
         "parameters": {
-            "clientId": "<CLIENT_ID>",
-            "organizationId": "<ORGANIZATION_ID>",
-            "shortCode": "<SHORT_CODE>",
-            "siteId": "<SITE_ID>"
+            "clientId": "c9c45bfd-0ed3-4aa2-9971-40f88962b836",
+            "organizationId": "f_ecom_zzrf_001",
+            "shortCode": "8o7m175y",
+            "siteId": "RefArchGlobal"
         }
     },
-    "defaultSite": "<SITE_ID>",
+    "defaultSite": "RefArchGlobal",
+    "defaultLocale": "en-US",
     "einsteinAPI": {
         "host": "https://api.cquotient.com",
-        "einsteinId": "<EINSTEIN_ID>",
-        "siteId": "<EINSTEIN_SITE_ID>",
+        "einsteinId": "1ea06c6e-c936-4324-bcf0-fada93f83bb1",
+        "siteId": "aaij-MobileFirst",
         "isProduction": false
     },
+    "search": {
```

Comment:
I listed search constants into it owns config because these search constants are likely to be used in a function outside of the PLP context. It is possible to use these values in different pages (when needed).